### PR TITLE
Added submit button.

### PR DESCRIPTION
**Pull request by artem-davidovich**

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -48,6 +104,15 @@ function Header() {
 					))}
 				</select>
 			)}
+			<button
+				disabled={requestsExist}
```
Comment (rzvdv) [inline src/components/Header.tsx:108]: disabled={!requestsExist}

**Comment by rzvdv** [comment]
I'm not sure if box updating is working correctly(in BoxUpdater.saveBoxChanges), i've started doing it a month ago, and i believe i haven't finished it. someone needs to revisit it and make sure everything works as intended before submitting this pr

File: src/components/Header.tsx [hunk 2]
```diff
@@ -28,13 +46,51 @@ const useStyles = createUseStyles({
...
+		display: 'none',
+	},
 });
 
 function Header() {
 	const schemaStore = useSchemaStore();
 
 	const classes = useStyles();
 
+	const requestsExist = computed(() => schemaStore.requestsStore.preparedRequests.length > 0).get();
```
Comment (rzvdv) [inline src/components/Header.tsx:88]: i think it's better to have this computed property in requestsStore

File: src/components/Header.tsx [hunk 3]
```diff
@@ -28,13 +46,51 @@ const useStyles = createUseStyles({
...
+
+	const submitChanges = () => {
+		schemaStore.requestsStore.saveChanges()
```
Comment (rzvdv) [inline src/components/Header.tsx:91]: we can just pass schemaStore.requestsStore.saveChanges to onClick button handler

**Review by YourWeebyNeighbor — APPROVED** [review]

**Pull request merged**